app/src/main/java/org/mozilla/focus/urlinput/UrlInputFragment.java

```diff
@@ -241,7 +241,7 @@ public void setSuggestions(@Nullable List<CharSequence> texts) {
             return;
         }
 
-        final String searchKey = urlView.getOriginalText().trim();
+        final String searchKey = urlView.getOriginalText().trim().toLowerCase();
```

Comment:
Looks like a good solution to me. I've got some other things in my mind, though:

A foreseeable situation I can see is this will not work when the user is using Turkish locale. which has a special lowercase of I : ı https://en.wikipedia.org/wiki/Dotted_and_dotless_I

so I would recommend:
final String searchKey = urlView.getOriginalText().trim().toLowerCase(Locale.US);

Another situation is, I believe some search suggestions are provided in capital: I haven't found an example in Google's suggestions but you can try "CD" for duckduckgo, this solution also break under suggestions like this. I'm not sure if lowering every suggestion sounds like a good idea. I'll leave it to the reviewer to decide :P @walkingice can you help review this?

Reply:
Yeah, I'm not that knowledgeable in regard to locale / localization, so, your input it highly appreciated. I'll wait what the reviewer said and will follow up accordingly...

Reply:
Let's first start with lowering the both since wikipedia search suggestions are always capitalized. I'll take a look at if there is any performance issue here. Since we're toLowerCase()ing both strings, I guess the locale would be irrelevant.

Reply:
I'm not sure by "lowering the both", can you elaborate?

Reply:
In:

`str.indexOf(searchKey);`

do 

`str.toLowerCase().indexOf(searchKey);` instead